did_core/did_methods/did_resolver_sov/src/resolution/utils.rs

```diff
@@ -53,6 +53,15 @@ fn unix_to_datetime(posix_timestamp: i64) -> Option<DateTime<Utc>> {
         .map(|date_time| DateTime::<Utc>::from_naive_utc_and_offset(date_time, Utc))
 }
 
+fn expand_abbreviated_verkey(did: &str, verkey: &str) -> String {
+    if verkey.starts_with('~') {
+        let did_public_key_part = &did[8..];
+        format!("{}{}", did_public_key_part, &verkey[1..])
+    } else {
+        verkey.to_string()
+    }
+}
+
```

Comment:
this won't exactly work; have a dig around some of the links in the original ticket. Here's a test vector to play with:

* nym/did: "7Sqc3ne5NfUVxMTrHahxz3"
* shortened verkey: "~DczaFTexiEYv5abkEUZeZt"
* expected (expanded) verkey: "4WkksEAXsewRbDYDz66aTdjtVF2LBxbqEMyF2WEjTBKk"